### PR TITLE
Switch Phabricator to use service accounts.

### PR DESCRIPTION
**Pull request by brendandburns**
@fgrzadkowski @erictune @lavalamp

### CODE REVIEW

**Comment by brendandburns** [comment]
Partially addresses #7444

File: examples/phabricator/cloudsql-authenticator/run.sh [hunk 1]
```diff
@@ -18,10 +18,11 @@
 #       should only send updates if something changes. We should be able to do
 #       this by comparing pod creation time with the last scan time.
 while true; do
-  hostport="${KUBERNETES_RO_SERVICE_HOST}:${KUBERNETES_RO_SERVICE_PORT}"
+  hostport="https://kubernetes.default.cluster.local"
+  token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
   path="api/v1beta1/pods"
```
Comment (lavalamp) [inline examples/phabricator/cloudsql-authenticator/run.sh]: Can you fix this while you're here?

File: examples/phabricator/cloudsql-authenticator/run.sh [hunk 2]
```diff
@@ -18,10 +18,11 @@
...
   query="labels=$SELECTOR"
-  ips_json=`curl ${hostport}/${path}?${query} 2>/dev/null | grep hostIP`
+  ips_json=`curl ${hostport}/${path}?${query} --insecure --header "Authorization: Bearer ${token}" 2>/dev/null | grep hostIP`
```
Comment (lavalamp) [inline examples/phabricator/cloudsql-authenticator/run.sh]: Add TODO about removing --insecure?

File: examples/phabricator/cloudsql-authenticator/run.sh (hunk 1 above)
Comment (brendandburns) [inline examples/phabricator/cloudsql-authenticator/run.sh]: done (I know you'd expect more changes, but just changing the path works...)

File: examples/phabricator/cloudsql-authenticator/run.sh (hunk 2 above)
Comment (brendandburns) [inline examples/phabricator/cloudsql-authenticator/run.sh]: done.

**Comment by brendandburns** [comment]
Comments addressed.  Please re-check.

Thanks!
--brendan

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**